methods.tex

```diff
@@ -34,18 +34,30 @@ \subsection{Performance Measurement}
   \label{eq:efficiency}
   E = \frac{S}{N}.
 \end{equation}
+
+All averages are defined as
+\begin{equation}
+\overline{t_{comp}} = \frac{1}{N}
+\sum_{\text{rank}=1}^{N}t_{\text{comp}} = \frac{1}{N}\sum_{\text{rank}=1}^{N}\sum_{\text{frame}=1}^{N_\text{b}}t_{\text{comp}}^{\text{frame}},
+\end{equation}
+\begin{equation}
+\overline{t_{\text{comm}}} = \frac{1}{N}\sum_{\text{rank}=1}^{N}t_{\text{communication}}
+\end{equation}
+\begin{equation}
+\overline{\tIO} = \frac{1}{N}\sum_{\text{rank}=1}^{N}\tIO = \frac{1}{N}\sum_{\text{rank}=1}^{N}\sum_{\text{frame}=1}^{N_{\text{b}}}t_{\text{I/O}}^{\text{frame}}
+\end{equation}
```

Comment:
Thanks, this is good.

(Now I realize that this is also in Table 3 but for readability it has to be in the main text.)

Reply:
This would fix #31